.github/workflows/test-build-docker-image.yml

```
@@ -240,7 +240,7 @@ jobs:
 
       # This step triggers an external workflow for automated analysis of Cypress test runs.
       - name: Invoke Automated analysis workflow
-        if: ( always() && github.ref == 'refs/heads/release' && github.run_attempt <= 2)
+        if: (always() && (github.ref == ‘refs/heads/release’ || github.ref == ‘refs/heads/master’) && github.run_attempt <= 2)
```

actionlint (1.7.4)

got unexpected character '‘' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)